frontend/packages/metal3-plugin/src/components/dashboard.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Dashboard, DashboardGrid } from '@console/internal/components/dashboard';
+import { HealthCard } from '@console/internal/components/dashboards-page/overview-dashboard/health-card';
+
+export const HostDashboard: React.FC<{}> = () => {
```

Comment:
Since the dashboard is going to consist of several custom card components, it may be good to put this file into `components/dashboard/index.tsx` and the cards will live in `components/dashboard/SomeCard.tsx`